main.go

```diff
@@ -60,12 +62,30 @@ func YarpNarpHandler(res http.ResponseWriter, req *http.Request) {
 
 func handleGet(res http.ResponseWriter, req *http.Request) {
 	z := Zusage{}
-	if cookie, _ := req.Cookie("nick"); cookie != nil {
-		z.Nick = cookie.Value
-	}
-	z = GetZusage(z.Nick)
-	if cookie, _ := req.Cookie("kommentar"); cookie != nil {
-		z.Kommentar = cookie.Value
+
+	if c, _ := req.Cookie("yarpnarp"); c != nil {
+		keks, err := base64.StdEncoding.DecodeString(c.Value)
+		if err != nil {
+			log.Println(err)
```

Comment:
You need to write *something* to the http-connection (i.e. do http.Error(…) here)